### PR TITLE
INTMDB-154: Deprecation for provider_encrypt_ebs_volume 

### PR DESCRIPTION
**Pull request by leofigy**
## Description

Now by default all EBS volumes will be encrypted by default, so the option to disable it is being deprecated. 

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [X] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X] I have read the Terraform contribution guidelines
- [X] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [X] I have added any necessary documentation (if appropriate)
- [X] I have run make fmt and formatted my code

### CODE REVIEW

**Comment by themantissa** [comment]
Just to note "Now by default all EBS volumes will be encrypted by default, so the option to disable it is being deprecated." - This has always been the case but I think it the past one could set it to false and that is no longer the case.  Just an important bit of clarification for anyone looking

File: mongodbatlas/resource_mongodbatlas_cluster.go [hunk 1]
```diff
@@ -168,8 +168,13 @@ func resourceMongoDBAtlasCluster() *schema.Resource {
 				Computed: true,
 			},
 			"provider_encrypt_ebs_volume": {
+				Type:       schema.TypeBool,
+				Optional:   true,
+				Deprecated: "All EBS volumes are now encrypted by default, no option to have encryption disabled",
```
Comment (themantissa) [inline mongodbatlas/resource_mongodbatlas_cluster.go]: Let's change this a bit to reduce confusion: 
"All EBS volumes are encrypted by default, the option to disable encryption has been removed",

**Review by themantissa — CHANGES_REQUESTED** [review]
Really nice, just one suggested wording change on the deprecation notice and this is ready.  Thank you!

**Review by themantissa — APPROVED** [review]
Thanks for the small fix!  LGTM!

**Review by coderGo93 — APPROVED** [review]
If all tests works then LGTM!

**Pull request merged**